ezpublish/config/config.yml

```diff
@@ -82,6 +82,8 @@ framework:
 twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
+    globals:
+        contactLocationId: %app.home.contact_location_id%
```

Comment:
Can't say that I'm a big fan of a global here. There are probably better ways to do this, but at least it's something.

But would it make sense to make that global var a hash, since we're most likely gonna have more than one location id in there ?